src/components/todolist/TodoList.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Todo from '../todo/Todo';
+import './TodoList.css';
+
+const TodoList = ({ usersInfo }) => (
+  <ul className="list">
+    {usersInfo.map(oneOfUsersInfo => (
+      <li className="list__userInfo" key={oneOfUsersInfo.id}>
+        <Todo userInfo={oneOfUsersInfo} />
+      </li>
```

Comment:
```suggestion
      <Todo userInfo={oneOfUsersInfo} key={oneOfUsersInfo.id} />
```
li перенеси внутрь компонента Todo